api/src/services/roles.ts

```diff
@@ -27,41 +27,116 @@ export class RolesService extends ItemsService {
 		}
 	}
 
-	private async checkForOtherAdminUsers(key: PrimaryKey, users: Alterations | Item[]): Promise<void> {
+	private async checkForOtherAdminUsers(
```

Comment:
First: This does fix the issue on my end! Good job! <3 :smile: 

Second: I want to apologize if I go too deep into subjective nitpicky territory here but I feel the usage of this check could be made more obvious. For example when I read the function name for the first time I didnt even understand what its supposed to do. 

- `check` well, check for what exactly? Existence I guess? But if thats the case then returning a boolean sounds more reasonable to me for example `hasRemainingAdmins` or `hasAtLeastOneAdmin` because what does "other" mean? Feels ambiguous
- Looking at the function its basically an `assert` since we dont return anything, we just throw so something like `assertAtLeastOneAdmin` feels clearer to read because assertions imply thrown exceptions

This would make the usage then a lot clearer in my opinion too.

Reply:
Since we already have the `checkForOtherAdminRoles` function which "checks whether there are other admin roles" i dont mind this naming